server/parsing_lib/log_parser.py

```diff
@@ -8,61 +8,128 @@
 
 
 class LogParser(object):
-    """ Handles parsing of all logs from the Mobile API. """
+    """ Handles parsing of all logs from the Mobile API.
+
+    Attributes:
+        parser_info: info required by the parser that is specific to each OS
+    """
+    parser_info = {
```

Comment:
Looking through the rest, I think that would be better. It will make this class a lot cleaner since it would be just the parsing logic and not convoluted with OS checks. Things like the HTML could also be put into baseParse so this class would be very short and concise